content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -5,53 +5,139 @@ weight = 50
 
-#### Authoring pipelines to use workload identity
+#### Authoring pipelines to use Workload Identity
 
-Pipelines don't need any change to authenticate to GCP, it will use the GSA bound to `pipeline-runner` KSA transparently.
+Pipelines don't need any specific changes to authenticate to Google Cloud. With Workload Identity, pipelines run as the Google service account that is bound to the KSA.
 
-However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove `use_gcp_secret` usage to use the bound GSA.
-You can also continue to use `use_gcp_secret` in a cluster with workload identity enabled, but pipeline steps with `use_gcp_secret` will use the GSA corresponding to the secret provided.
+However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove the `use_gcp_secret` usage to use the bound GSA.
+You can also continue to use `use_gcp_secret` in a cluster with Workload Identity enabled, but pipeline steps with `use_gcp_secret` will use the GSA corresponding to the secret provided.
 
-#### Cluster setup to use workload identity for Pipelines Standalone
+#### Cluster setup to use Workload Identity for Pipelines Standalone or AI Platform Pipelines
 
-**After you enabled workload identity**, you need to bind workload identities for KSAs used by [Pipelines Standalone](/docs/pipelines/installation/overview/#kubeflow-pipelines-standalone).
-The following helper bash scripts bind workload identities for KSAs provided by Pipelines Standalone:
+##### 1. Create your cluster with Workload Identity enabled
+
+* In Google Cloud Console UI, you can enable Workload Identity in `Create a Kubernetes cluster -> Security -> Enable Workload Identity` like the following:
+<img src="/docs/images/pipelines/gke-enable-workload-identity.png">
+
+* Using `gcloud` CLI, you can enable it with:
+```bash
+gcloud beta container clusters create cluster-name \
+  --release-channel regular \
+  --workload-pool=project-id.svc.id.goog
+```
+
+References:
+
+* [Enable Workload Identity on a new cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster)
+
+* [Enable Workload Identity on an existing cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_an_existing_cluster)
+
+##### 2. Deploy Kubeflow Pipelines
+Deploy via [Pipelines Standalone](/docs/pipelines/installation/overview/#kubeflow-pipelines-standalone) or [AI Platform Pipelines](/docs/pipelines/installation/overview#gcp-hosted-ml-pipelines) as usual.
+
```

Comment:
same as previous comment,
1. remove AI Platform Pipelines UI path here for now.
2. "Pipelines Standalone" to "Pipelines Standalone CLI"

Reply:
Why do we remove AI Platform Pipelines UI path here for now.?

Reply:
Enabled WI after a MKP-UI based installation is not a designed behavior. It would largely confuse user. MKP-UI is more for entry-level user who can quickly play KFP without these additional setup.